recipe/meta.yaml

```
@@ -32,29 +32,31 @@ requirements:
     - cachey >=0.2.1
     - certifi >=2020.6.20
     - dask >=2.15.0
-    - imageio >=2.5.0
+    - imageio >=2.5.0,!=2.11.0
     - importlib-metadata >=1.5.0  # not needed for py>37 but keeping for noarch
     - jsonschema >=3.2.0
-    - magicgui >=0.3.0
+    - magicgui >=0.3.3
     - napari-console >=0.0.4
     - napari-plugin-engine >=0.1.9
     - napari-svg >=0.1.4
+    - npe2 >=0.1.1
     - numpy >=1.18.5
     - numpydoc >=0.9.2
+    - pandas >=1.1.0
     - pillow
```

Same. [Upstream](https://github.com/napari/napari/blob/main/setup.cfg#L57) says:

```suggestion
    - pillow !=7.1.0,!=7.1.1
```

---

it's true... that ones been there for a while.  should have been here

---

Do you think we need to repodata-patch the previous releases in that case?

---

I dunno.  I do think they were somewhat severe (breaking our builtin image reader https://github.com/napari/napari/issues/1098, https://github.com/napari/napari/pull/1099, and https://github.com/python-pillow/Pillow/issues/4509), but it was also very short-lived... (with 7.1.2 coming out quickly).  I'll defer to you/others on that one

---

Hm, it's a corner case but it should be fixed. If for some reason another package is pinning on pillow 7.1.0 and it's installed long napari, it could force a downgrade to find a "theoretically compatible" napari that does support that pillow version. I'll tag you in the PR when ready. Are all releases affected?

---

See https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/214

---

🤯

---

i haven't done a thorough check, but from my understanding of the issue, I think it's safer to assume that all versions are affected